Formula/bioformats53.rb

```diff
@@ -1,8 +1,8 @@
 class Bioformats53 < Formula
   desc "Library for reading proprietary image file formats"
   homepage "http://www.openmicroscopy.org/site/products/bio-formats"
-  url "http://downloads.openmicroscopy.org/bio-formats/5.3.2/artifacts/bioformats-5.3.2.zip"
-  sha256 "f9dde792dc24f7cf27e558dda8d0f69dddfde4d41a4339f79e5bb5fd11793ea8"
```

Comment:
Redownloaded the zip file and re ran the command and I am now getting the sha from the CI job. Not sure why they would be different though